spec/controllers/bets_controller_spec.rb

```diff
@@ -20,5 +20,186 @@
 
       expect(create_bet).to have_received(:save)
     end
+
+    it 'calls CreateBet form with correct attributes' do
```

Comment:
There is no point of creating three separate tests for:
- 'calls CreateBet form object'
- 'calls CreateBet form with correct attributes'
- 'create Bet with correct attributes'

One 'calls BetForm with correct attributes' will be enough. Same for update - also form object name is BetForm, not CreateBet :)